### PR TITLE
Migrate UUIDField uuid in Parameter model

### PR DESCRIPTION
**Pull request by hackdna**
Toward #2592

### CODE REVIEW

File: refinery/tool_manager/models.py [hunk 1]
```diff
@@ -509,8 +508,7 @@ class Meta:
         )
 
     def get_container_input_dict(self):
-        """
-        Create a dictionary containing information that Dockerized
+        """Create a dictionary containing information that Dockerized
```
Comment (jkmarx) [inline refinery/tool_manager/models.py]: Why the format change?

File: refinery/tool_manager/tests.py [hunk 2]
```diff
@@ -255,10 +255,8 @@ def setUp(self):
         self.tool_defs_url_root = '/api/v2/tool_definitions/'
 
         self.mock_parameter = ParameterFactory(
-            name="Test Param",
-            description="Test Param Description",
-            value_type=Parameter.STRING,
-            default_value="Coffee"
+            name="Test Param", description="Test Param Description",
```
Comment (jkmarx) [inline refinery/tool_manager/tests.py]: The left format is easier to read. Why the change?  https://github.com/refinery-platform/refinery-platform/wiki/coding-style-guide#source-formatting

File: refinery/tool_manager/models.py (hunk 1 above)
Comment (hackdna) [reply refinery/tool_manager/models.py]: Unnecessary newline (https://www.python.org/dev/peps/pep-0257/#id17)

Comment (jkmarx) [reply refinery/tool_manager/models.py]: It'd be great to automate any rules we want to enforce. I personally find the left easier to read.

File: refinery/tool_manager/tests.py (hunk 2 above)
Comment (hackdna) [reply refinery/tool_manager/tests.py]: It's customary in Python to list function arguments with no extra new lines (example: https://www.python.org/dev/peps/pep-0008/#indentation). Also, unnecessary newlines make functions/methods longer and more difficult to read (i.e., to fit on one screen).
